packages/core/src/domain/configuration/transportConfiguration.ts

```diff
@@ -77,7 +77,7 @@ function computeReplicaConfiguration(
   buildEnv: BuildEnv,
   intakeEndpoints: string[]
 ): ReplicaConfiguration | undefined {
-  if (buildEnv.buildMode !== BuildMode.STAGING || initConfiguration.replica === undefined) {
+  if (!initConfiguration.replica) {
```

Comment:
After checking on prod I saw that replica is set to "null" therefore I replaced  `initConfiguration.replica === undefined` by `!initConfiguration.replica`.